Графика для всех страниц

irina261193 opened the pull request:
---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/35/user/1890517/tasks/7)

irina261193 committed:
оптимизация графики

исправила альты картинок, вытащила svg файлы, оптимизировала svg, png и jpeg

Pakistanka reviewed (changes requested):
1. Логотипы это всегда svg - http://joxi.ru/V2V7Mg0uBpVpQA
2. Это тоже svg - http://joxi.ru/xAeo3OZcM363Nm
3. Это тоже svg - http://joxi.ru/DrlWy9xHKYnYPA

irina261193 committed:
исправление изображений

исправила формат изображений, изображения оптимизировала и снова прописала в коде
